calcom/cal.com
# changed translations on booking page to better match the scenario.

## What does this PR do?



From a Japanese perspective, after booking an event the booking successful page displayed has a Japanese language that needs to be changed. Currently, it seems to be translated without context. I have fixed a few areas where the language is better suited. 

Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change






## How should this be tested?


et your browser language to Japanese, and book for an event. 


- Are there environment variables that should be set?
- no
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- set your browser language to Japanese, and book for an event
- Any other important info that could help to test that PR

## Mandatory Tasks

- [ X] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist


- I haven't added tests that prove my fix is effective or that my feature works

![スクリーンショット 2023-07-27 19 37 55](https://github.com/calcom/cal.com/assets/16210881/9da7c8a3-7692-4fb7-b5df-c3639549a449)
